fedimint-core/src/session_outcome.rs

```diff
@@ -15,11 +15,16 @@ pub struct AcceptedItem {
     pub peer: PeerId,
 }
 
-/// All items ordered in a session that have been accepted by Fedimint Consensus
-/// are recorded in the corresponding block. A running Federation produces a
-/// [SessionOutcome] roughly every five minutes.  Therefore, just like in
-/// Bitcoin, a [SessionOutcome] might be empty if no items are ordered in that
-/// time or all ordered items are discarded by Fedimint Consensus.
+/// Items ordered in a single session that have been accepted by Fedimint
+/// consensus.
+///
+/// A running Federation produces a [SessionOutcome] roughly every five minutes.
```

Comment:
[nit] Without high latency, I believe we should expect a session outcome every 180s. It might be worth updating to `three minutes`, but can also see how "roughly" is close enough and leaves wiggle room for high latency networks.

@joschisan please correct me if I'm misunderstanding